WORKSPACE.bazel

```diff
@@ -12,6 +12,12 @@ http_archive(
     url = "https://github.com/bazelbuild/rules_jvm_external/archive/%s.zip" % RULES_JVM_EXTERNAL_TAG,
 )
 
+load("@rules_jvm_external//:repositories.bzl", "rules_jvm_external_deps")
```

Comment:
note for review: without these two load+invocations, `bazel build //...` no longer works, since the `java_export` rule declares some extra rules for deployment that we actually don't need. If there is a way to suppress those from existing, or if we aren't bothered that `//...` can't build, this can be removed.